internal/dev_server/adapters/mocks/utils.go

```diff
@@ -0,0 +1,17 @@
+package mocks
+
+import (
+	"context"
+
+	"github.com/launchdarkly/ldcli/internal/dev_server/adapters"
+	"go.uber.org/mock/gomock"
+)
+
+func WithMockApiAndSdk(ctx context.Context, controller *gomock.Controller) (context.Context, *MockApi, *MockSdk) {
```

Comment:
Fancy!

Reply:
Maybe backport this [here](https://github.com/launchdarkly/ldcli/blob/moonshot-ld-dev-server/internal/dev_server/sdk/go_sdk_test.go#L44-L48)